package.json

```diff
@@ -53,20 +53,24 @@
     "eslint-plugin-react": "^7.10.0",
     "eslint-plugin-standard": "^3.1.0",
     "gh-pages": "^1.2.0",
-    "jest": "24.8.0",
+    "jest": "^23.6.0",
     "react": "^16.4.1",
     "react-dom": "^16.4.1",
-    "react-scripts": "^3.1.1",
+    "react-scripts": "^1.1.4",
     "react-test-renderer": "^16.6.3",
     "rollup": "^0.64.1",
     "rollup-plugin-babel": "^3.0.7",
     "rollup-plugin-commonjs": "^9.1.3",
     "rollup-plugin-node-resolve": "^3.3.0",
     "rollup-plugin-peer-deps-external": "^2.2.0",
-    "rollup-plugin-postcss": "^2.0.3",
+    "rollup-plugin-postcss": "^1.6.2",
     "rollup-plugin-url": "^1.4.0"
   },
   "files": [
     "dist"
-  ]
+  ],
+  "dependencies": {
+    "formik": "^1.5.1",
```

Comment:
where is this used ?

Reply:
sorry, I had not noticed that I added it in commit. I will delete it.